Assets/MixedRealityToolkit.Extensions/SpectatorView/Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView.asmdef

```diff
@@ -1,21 +1,20 @@
 {
-  "name": "Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView",
-  "references": [
-    "Microsoft.MixedReality.Toolkit.Extensions.Experimental.MarkerDetection",
-    "Microsoft.MixedReality.Toolkit.Extensions.PhotoCapture",
-    "Microsoft.MixedReality.Toolkit.Extensions.Experimental.ScreenRecording",
-    "Microsoft.MixedReality.Toolkit.Extensions.Experimental.Sharing",
-    "Unity.XR.ARFoundation",
```

Comment:
Including this by default gave me some issues with GoogleARCore, we should have this in instructions and tell user to then go set "AR_FOUNDATION" scripting define symbol.